packages/js/product-editor/src/hooks/use-feedback-bar/use-feedback-bar.ts

```diff
@@ -64,9 +47,26 @@ export const useFeedbackBar = () => {
 		} );
 	};
 
+	const fetchShowFeedbackBarOption = useCallback( () => {
+		return resolveSelect( OPTIONS_STORE_NAME )
+			.getOption< string >( PRODUCT_EDITOR_SHOW_FEEDBACK_BAR_OPTION_NAME )
+			.then( ( showFeedbackBarOption ) => {
+				if (
+					window.wcTracks?.isEnabled &&
+					showFeedbackBarOption === 'yes'
+				) {
+					setShouldShowFeedbackBar( true );
```

Comment:
This is a bit confusing. It sets "shouldShowFeedbackBar", but also hides the feedback bar at the same time. The concept is a little hard to understand.

Reply:
Good call @nathanss! I refactored the code a little bit to make it easier to read, in the commit 5010378.

Could you take another look at this PR?